### DISCUSSION TITLE
Nova for other languages than English?

### DISCUSSION THREAD

**Discussion by willemmulder** [deepgram/community#348] [General help] [answered]
We're a dutch company using Deepgram for transcription, and we'd love to be part of the exciting new Nova models coming out :-D 
Where is this on the roadmap? Thanks!

**Comment by jjmaldonis** [comment, marked as answer]
Hey @willemmulder, yes we are building Nova 2 models for different languages. Dutch is on the list but currently we do not have an ETA for Dutch. I will forward your request to our team.

**Reply by willemmulder** [reply]
Thanks @jjmaldonis that's much appreciated! It's pretty amazing what you guys are delivering; keep up the work!

**Comment by cateas** [comment]
@jjmaldonis do you have an ETA for Spanish language for nova-2?

Thanks!

**Reply by jjmaldonis** [reply]
Hey @cateas, Spanish and French are planned to be available in November. We do not have an ETA for other languages at the moment.

**Reply by willemmulder** [reply]
Good news! Once you have an ETA for other languages, please keep us updated!
